### DISCUSSION TITLE
All RTP channels are not working in Kodi (but they do in VLC)

### DISCUSSION THREAD

**Discussion by cssobral2013** [iptv-org/community#140] [Show and tell]
### Stream Title

RTP, RTP2, RTP Internacional, ... (any RTP channel)

### What happened to the stream?

Other

### Playlist

https://iptv-org.github.io/iptv/index.m3u

### Notes

This issue only displays in Kodi. In other players, such as VLC, it is not displayed

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

**Comment by BellezaEmporium** [comment]
Invoking @Sphinxroot , as he's our Kodi specialist.

**Comment by Sphinxroot** [comment]
hi  cssobral2013

all RTP link  in IPTV-ORG list 

are coded ,as exemple :
#EXTINF:-1 tvg-id="RTPInternacional.pt" status="online" user-agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0",RTP Internacional (504p) [Not 24/7]
#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
https://streaming-live.rtp.pt/liverepeater/smil:rtpi.smil/playlist.m3u8

to properly work in KODI  they must be  coded as this :
#EXTINF:-1 tvg-id="RTPInternacional.pt" ,RTP Internacional (504p) [Not 24/7]
https://streaming-live.rtp.pt/liverepeater/smil:rtpi.smil/playlist.m3u8|User-Agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36

you can  built  your own  m3u playlist  and host it here on github ,  and use playlistloader  addon   leia(18.9) or matrix (19.x) ,  and  add  in remote mode in the addons  , so if you modified the m3u  it will auto update in the addons

hope it help you :)

**Comment by Sphinxroot** [comment]
something is also weird .. a couple of the them have  status="error"   ,but they do work
